MeNext/Layout/SearchView.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Xamarin.Forms;
+
+
+namespace MeNext
+{
+	public class SearchView : ContentPage
+	{
+		private SearchBar searchBar;
+		private Label resultsLabel;
+		private SongListView songList;
+
+		public SearchView()
+		{
+			// TODO: clean up this testing code
+			this.Title = "Search";
+			NavigationPage.SetHasNavigationBar(this, false);
+			Song selectedSong = null;
+
+			resultsLabel = new Label
+			{
+				Text = "Results will appear below.",
```

Comment:
I feel like the user can guess this from the visual queues provided / will figure it out once they start searching